### PR TITLE
Look for krb5_klog_syslog in libkadm5srv.

### PR DESCRIPTION
**Pull request by FransUrbo**
Closes: #2 

Also, I'm running `autoconf` v2.69, and it complained about the `AC_SUBST` line:

```
$ autoreconf -fiv
autoreconf: Entering directory `.'
autoreconf: configure.in: not using Gettext
autoreconf: running: aclocal --force
aclocal: warning: autoconf input should be named 'configure.ac', not 'configure.in'
configure.in:293: error: AC_SUBST: `SUBDIR LDSH LDSHFLAGS PICFLAG SOEXT SOEXTVER LIBINC MKSO' is not a valid shell variable name
configure.in:293: the top level
autom4te: /usr/bin/m4 failed with exit status: 1
aclocal: error: echo failed with exit status: 1
autoreconf: aclocal failed with exit status: 1
```

Not sure why, but according to the manual for *autoconf*, it only takes one parameter (or two, if the second is the value): http://www.gnu.org/savannah-checkouts/gnu/autoconf/manual/autoconf-2.69/html_node/Setting-Output-Variables.html#Setting-Output-Variables

If you want me to do that as a separate patch, let me know.

### CODE REVIEW

**Comment by grawity** [comment]
Single PR is fine, but it would still be nicer to have both fixes as two separate commits.

I guess autoconf has become stricter recently – the last time I've built this was half a year ago and didn't have any problems back then. (kcrap-lnf is not my project, just a fork of a dead fork of the dead original kcrap.)

**Comment by FransUrbo** [comment]
LOL, no problem.

File: configure.in [hunk 1]
```diff
@@ -249,7 +255,7 @@ if [[ "$build_server" = 1 ]]; then
     AC_MSG_RESULT(not 8-argument)
   ])
 
-  LIBS="$save_LIBS"
+#  LIBS="$save_LIBS"
```
Comment (grawity) [inline configure.in]: I guess this (and the above `save_LIBS=$LIBS`) should be removed entirely, then?

Comment (FransUrbo) [inline configure.in]: Ah, yes.

**Pull request merged**